src/main/java/com/work/management/converter/ListConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.work.management.converter;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.Arrays;
+import java.util.List;
+
+@Converter
+public class ListConverter implements AttributeConverter<List<String>,String> {
```

Comment:
Try accepting ImmutableList in all function parameters and in the class declaration also.
AtrributeConverter<ImmutableList<String>, String>
..there should be a space after comma.